paasta_tools/cli/cmds/local_run.py

```diff
@@ -519,6 +538,26 @@ def check_if_port_free(port):
     return True
 
 
+def decrypt_secret_environment_variables(
+    secret_provider_name,
+    environment,
+    soa_dir,
+    service_name,
+    cluster_name,
+    secret_provider_kwargs,
+):
+    ecosystem = secret_provider_kwargs['vault_cluster_config'].get(cluster_name, 'devc')
```

Comment:
Hmm. Yuck :(
Why does this need to be here, can't the lower-level secret provider "know" where it is?

Reply:
Not ideal is it :-(

The reason this exists is we need to map between paasta cluster and vault cluster. There are some paasta clusters that don't have a local vault cluster and we need to know which one to talk to in those cases (and which secret ciphertext to read from yelpsoa-configs.

Are you thinking that this map should be a config file that belongs to vault_tools instead of part of PaaSTAs config? I think maybe that would be better, it's really just here because it was an "easy" place to put it for now.